packages/cozy-doctypes/src/banking/BankTransaction.spec.js

```diff
@@ -65,7 +65,7 @@ describe('getMissedTransactions', () => {
     expect(missedTransactions).toEqual([newTransactions[1]])
   })
 
-  it('should return transactions with an already existing identifier, if there are more new than existing', () => {
+  xit('should return transactions with an already existing identifier, if there are more new than existing', () => {
```

Comment:
I am not sure that we use this thing ? Do you think it is still relevant @drazik ?